Automattic/pocket-casts-ios
# End of Year: design tweaks

| 📘 Project: #376 |
|:---:|

This PR address different design changes:

* How the isometric artwork is displayed (plus update the positioning)
* Change the position of the close button and share (and update its icons)
* Change some fonts and padding
* Add breakline in some texts

### iPhone 8

https://user-images.githubusercontent.com/7040243/202212533-cba34015-e343-48c4-8da2-6d7b844db8bf.mp4

### iPhone 14

https://user-images.githubusercontent.com/7040243/202212652-4873d230-d959-41f7-91bd-03e87d6a944b.mp4

### iPhone 14 Pro Max

https://user-images.githubusercontent.com/7040243/202213281-3a8d2a69-751d-44f5-8a63-e053d9ddec83.mp4

## To test

Run the app and check the stories on 3 different devices (big, medium, and small). Eg.: iPhone 8, iPhone 14, iPhone 14 Pro Max.

## Checklist
